src/ray/rpc/retryable_grpc_client.h

```diff
@@ -201,13 +205,13 @@ class RetryableGrpcClient : public std::enable_shared_from_this<RetryableGrpcCli
   // to prevent OOM.
   const uint64_t max_pending_requests_bytes_;
   const uint64_t check_channel_status_interval_milliseconds_;
-  const uint64_t server_unavailable_timeout_seconds_;
-  // After the server is unavailable for server_unavailable_timeout_seconds_,
-  // this callback will be called.
+  const uint64_t server_unavailable_base_timeout_seconds_;
+  // After the server is unavailable for an exponential backoff period, this callback will
```

Comment:
this comment is a little misleading, makes it sound like it'll be called once after the full exp backoff sequence. instead something like:

> This callback will be called periodically while the server has been unavailable. The period between checking the status of the channel and calling this callback increases with exponential backoff.